### DISCUSSION TITLE
OBS New versions don't remember logins

### DISCUSSION THREAD

**Discussion by tmyrdal** [obsproject/obs-studio#5941] [General]
When you use Widgets in obs let say, the restream options inside Stream options if you select restream then you log in and stuf like that and you are logged in, then if you exit the IBS studio and next time you start it you need to log in again, it doesn't remember your login.

This also is regarding if you add a Browser scourge in the "Sources" list then its not logged in next time, so you hafto login again next time you start OBS studio.

On the OBS studio the 27 version you never hafto log in again after you are logged in it remembers that you are logged in.

Way don't this work with the new OBS Studio? the Release Candidate don't work there either it doesn't remember logins.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for assistance, specifically the #beta-testing channel in our Discord as the best place.

Thank you!
